package.json

```diff
@@ -292,6 +297,7 @@
 		"vscode-debugadapter-testsupport": "^1.40.3"
 	},
 	"dependencies": {
+		"@vscode/test-electron": "^2.3.3",
```

Comment:
Shouldn't this be moved up to devDependencies?

Reply:
I haven't _checked_ if `devDependencies` is enough, but I still think so.

Reply:
Sorry! I didn't understand. Can you give me more information.

Reply:
OK! I will correct. Thanks.

Reply:
This one is in error, we do need the `dependencies` - but both new "test" ones should be included in `devDependencies`.